zip-0227.rst

```diff
@@ -96,9 +106,47 @@ Issuance Key Derivation
 ```````````````````````
 
 The issuance master key is generated by choosing a bit sequence uniformly at random from :math:`\mathbb{B}^{\mathbb{Y}[32]}`, like the Orchard spending key [#protocol-orchardkeycomponents]_.
-In :math:`\mathsf{zcashd}`, this key is derived in a manner similar to the generation of the Orchard master key in ZIP 32 [#zip-0032-orchard-master]_, as detailed below:
 
-- Details TBD
+Issuance master key derivation for hierarchical deterministic wallets
+''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''' 
+
+In ``zcashd``, this key is derived in a manner similar to the generation of the Orchard master key in ZIP 32 [#zip-0032-orchard-master]_, as detailed in this section.
+
+Let :math:`S` be a seed byte sequence of a chosen length, which MUST be at least 32 bytes and at most 252 bytes.
+
+- Calculate :math:`I = \textsf{BLAKE2b-512}(\texttt{"ZIP32ZSAIssuance"},S)`.
+- Split :math:`I` into two 32-byte sequences :math:`I_L` and :math:`I_R`.
+- Return :math:`(\mathsf{sk}_m, \mathsf{c}_m) := (I_L, I_R)` as the master extended issuance key, :math:`m_{\mathsf{Issuance}}`.
+
+As in ZIP 32 for Orchard [#zip-0032-orchard-child-key-derivation]_, we only use hardened child key derivation for issuance master key. 
+We define the following function for child key derivation:
+
+:math:`\mathsf{CDKik}((\mathsf{sk}_{par},\mathsf{c}_{par}),i) \to (\mathsf{sk}_i, \mathsf{c}_i)`
+
+- Check whether :math:`i \geq 2^{31}` (i.e. whether the child is a hardened key)
+
+  - If so (hardened child): Let :math:`I = \mathsf{PRF^{expand}}(\mathsf{c}_{par}, [\mathtt{0xa1}] \| \mathsf{sk}_{par} \| \mathsf{I2LEOSP}_{32}(i))`.
+  - If not (non-hardened child): return failure.
+
+- Split :math:`I` into two 32-byte sequences :math:`I_L` and :math:`I_R`.
+- Set :math:`(\mathsf{sk}_i, \mathsf{c}_i) = (I_L, I_R)`.
```

Comment:
The `0xa1` "personalization" is the only thing changed from the Orchard definition of the CDKsk function. But this domain separation seems necessary, so it required defining a fresh function (that I named CDKik for clarity)